azure-quantum/azure/quantum/target/microsoft/qio/simulated_annealing.py

```diff
@@ -109,7 +109,7 @@ def from_target_status(
         :return: Target instance
         :rtype: Target
         """
-        if status.id.endswith("cpu"):
+        if status.id.endswith("cpu") or status.id.endswith("cpu.experimental"):
             platform = HardwarePlatform.CPU
         elif status.id.endswith("fpga"):
```

Comment:
Do we expect for there to be an fpga.experimental in the future as well? Maybe we should add it just in case?

Reply:
Currently ...fpga.experimental is not implemented and will raise unknown target error, which is correct behavior as of today.

Reply:
Sounds good!